app/lib/data/learn.ts

```diff
@@ -24,7 +24,7 @@ export default async function learn(_kwargs: Props) {
     vecdb: "main",
     filters: {
       language: language ? [language] : [],
-      doc_type: doc_type && Array.isArray(doc_type) ? doc_type : ["blog", "publications", "news"],
+      doc_type: doc_type && Array.isArray(doc_type) ? doc_type : ["blog", "publications", "news", "press release"],
```

Comment:
I am not sure we should include the press releases.

Reply:
Many of the time, country offices publishes blogs via press release pages. Press release contents are mostly richer than news. We have over 700 press release content in nlp and about 300 news articles. We could also in the background combine press release and news together, and only present one of the options to user but we fetch both.